muesli/web/templates/overview.pt

```diff
@@ -95,16 +90,8 @@
           </tr>
           <tr class="row text-center table-secondary mb-3">
             <td class="col">
-              <a role="button"
-                 class="btn btn-info"
-                tal:attributes="href request.route_path('lecture_view_points',
-                lecture_id=tutorial.lecture.id)">Details</a>
-            </td>
-            <td class="col">
-              <a role="button" class="btn btn-danger text-white" data-toggle="modal" data-target="#confirm_unsubscribe">
-                Austreten
-              </a>
-              <metal:main metal:use-macro="templates('Fragments/unsubscribe_modal.pt').macros['unsubscribe_modal']" />
```

Comment:
Why did you remove this? Should be totally fine to stay here with the modal implementation

Reply:
Again, UX considerations of mine: It didn't feel consistent to me that you can leave a tutorial on the front page whereas you have to go to the details page to switch to a different tutorial.

That, combined with the (as you remarked rather weak) argument of accidental click prevention, was my reason for the deletion.